client/components/diff-window.js

```diff
@@ -92,8 +92,11 @@ class DiffWindow extends React.Component {
         } catch (e) {
           return { added: [], removed: [] };
         }
-
-        diffs = diffWords(before.document.text, after.document.text);
+        if (before.document.text.length < 1000 && after.document.text.length < 1000) {
```

Comment:
This is 1k chars which is roughly 150 words or about a paragraph - could we maybe give them ~3 paragraphs before switching or is that still too slow?